CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ dgl_option(EXTERNAL_NANOFLANN_PATH "Path to use external nanoflann" OFF)
 dgl_option(EXTERNAL_METIS_PATH "Path to external metis" OFF)
 dgl_option(EXTERNAL_METIS_LIB_PATH "Path to external metis library" OFF)
 dgl_option(EXTERNAL_GKLIB_PATH "Path to external gklib" OFF)
+dgl_option(EXTERNAL_LIBURING_PATH "Path to external liburing" OFF)
```

Comment:
seems the purpose or usage of `EXTERNAL_XXX` is a bit different from you're using for liburing. `EXTERNAL_XXX` is for specify path from external instead of the default one under `third_party/xxx`. And what you want to achieve here is more like `USE_CUDA`.